### PR TITLE
Drawables for self test updated

### PR DESCRIPTION
**Pull request by rounaakk**
please hacktoberfest-accepted label to the PR if it is ok.

### CODE REVIEW

**Comment by jayskhatri** [comment]
Thanks @Rounak122 , for opening the pull request! 👏

I'll be reviewing your pull request thoroughly

If you liked the project show some love by Starring ⭐ this project.

File: app/google-services.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (jayskhatri) [inline app/google-services.json:1]: Hey @Rounak122 please remove this google-services.json file, as this should not be uploaded to github repo 😬

**Review by jayskhatri — CHANGES_REQUESTED** [review]
@Rounak122  you have deleted old illustrations rt? Correct me if I am wrong.

I am not getting where you have updated the new illustrations.

Everything is good, please delete the google-services.json.

PR will be merged after ur changes 👍

File: app/google-services.json (hunk 1 above)
Comment (Rounak122) [reply app/google-services.json:1]: @jayskhatri I have deleted old illustrations and renamed the new ones the same. I am removing google-services.json, I didn't knew that, thanks.

Comment (Rounak122) [reply app/google-services.json]: @jayskhatri I have done the changes. please check once.

**Review by jayskhatri — APPROVED** [review]
Good to go👍

Thanks for contributing @Rounak122 🙌

**Pull request merged**